src/mscorlib/corefx/System/Globalization/JapaneseCalendar.Win32.cs

```diff
@@ -7,12 +7,51 @@ namespace System.Globalization
 {
     public partial class JapaneseCalendar : Calendar
     {
-        public static int GetJapaneseEraCount()
+        private static EraInfo[] GetJapaneseEras()
         {
-            return WinRTInterop.Callbacks.GetJapaneseEraCount();
+            int erasCount = WinRTInterop.Callbacks.GetJapaneseEraCount();
```

Comment:
Short answer, Yes.  Due to risk aversion we don't want to touch the stuff in src/mscorlib/src, because that's the desktop code base.

We'll continue to evolve the version of the code that we ship with .NET Native (and cross plat CoreCLR).  Due to the SxS nature of the deployment vehicles we are less worried about accidental breaking changes.

Note that we've always had a split here, .NET Native and Desktop CLR have different implementations of the core globalization stuff because of the different runtimes between the two products.